src/makeRobot.js

```diff
@@ -38,7 +38,55 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goRight(move = 1) {
+      if (move === Math.abs(move)) {
+        this.coords.x += move;
+      }
+
+      return this;
+    },
```

Comment:
```suggestion
  const robot = {
    name,
    wheels,
    version,
    coords: {
      x: 0,
      y: 0,
    },

    get info() {
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
    },

    get location() {
      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
    },

    goRight(move = 1) {
      if (move === Math.abs(move)) {
        this.coords.x += move;
      }

      return this;
    },
```

Add empty line between methods